app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
```

Comment:
The `people` dictionary is defined as a class attribute. This means it will be shared across all instances of `Person`. If this is intended to store all `Person` instances globally, it's fine. Otherwise, consider using instance attributes.